### PR TITLE
Created Contact Us Page

### PR DESCRIPTION
**Pull request by mahek0620**
Issue No #54  - Implementatio of Contact Us Page
Screenshot 2024-05-27 224418

### CODE REVIEW

**Comment by aswinikalyan30** [comment]
@mahek0620 can you link all the issues you are solving here?
Also add screenshots for all changes made?

**Comment by mahek0620** [comment]
I have already uploaded screenshot above. 
Clicking the 'Contact Us' button will open a new page containing contact details such as phone number, email, and address, along with an attached form to fill the details. It allows users to offer feedback on the system, which can be valuable for making improvements and addressing user concerns. Users can report any technical problems they encounter, ensuring that issues are documented and can be resolved efficiently. It provides a direct channel for users, such as faculty members, students, and administrators, to seek assistance or report issues.
Note: For adding the social media links, a footer should be made on the website and social media links should be shared there instead of sharing in the Contact Us button. I will open a new issue and will fix that issue if you agree.

**Comment by aswinikalyan30** [comment]
Yes, I do understand that, I wanted you to add it for the reset email and password that you had implemented. Which is linked to another issue

**Comment by mahek0620** [comment]
image
image
Here is the screenshot of login page with Forgot Password option

**Comment by aswinikalyan30** [comment]
In issue #52 you mentioned you will implement a "sign in with google" button - Is that also added on this PR itself?

**Comment by mahek0620** [comment]
@aswinikalyan30 Thank you for your feedback. I will surely add the logo

**Comment by mahek0620** [comment]
can you please assign signin with google to one more person as I am facing difficulties in doing that

**Comment by aswinikalyan30** [comment]
Sure, will update that issue only for reset

**Comment by mahek0620** [comment]
@aswinikalyan30 I have added logo in the contactUs page

File: login.php [hunk 1]
```diff
@@ -4,157 +4,76 @@
...
-          <div class="input-group mb-3">
-            <input type="password" class="form-control" name="password" required placeholder="Password">
-            <div class="input-group-append">
-              <div class="input-group-text">
-                <span class="fas fa-lock" onclick="togglePasswordVisibility()"></span>
-                <span class="fas fa-eye" style="display:none;" onclick="togglePasswordVisibility()"></span>
-              </div>
+<body>
+    <h1 id="page-title" class="text-center">Login Page</h1>
+    <hr id="title_hr" class="mx-auto">
```
Comment (aswinikalyan30) [inline login.php:48]: Add logo (from prev (line 51)

File: login.php [hunk 2]
```diff
@@ -4,157 +4,76 @@
...
-            <div class="col-8">
-              <div class="icheck-primary">
-                <input type="checkbox" id="remember">
-                <label for="remember">
-                  Remember Me
-                </label>
-              </div>
+            <div class="input-field">
+                <label for="password" class="input-label">Password</label>
+                <input type="password" id="password" name="password" value="<?= htmlspecialchars($_POST['password'] ?? "", ENT_QUOTES) ?>" required="required">
```
Comment (aswinikalyan30) [inline login.php:69]: There was some code written by a prev contributor for toggle of password "fas fa-eye" - please don't scrape whatever was written before. I can see the different icons that were there are removed. It is okay to make additions to the file, this looks like is rebranded.

**Review by aswinikalyan30 — CHANGES_REQUESTED** [review]
Please update the login.php file , otherwise looks good!

**Comment by mahek0620** [comment]
@aswinikalyan30 as you mentioned that There was some code written by a prev contributor for toggle of password "fas fa-eye" - please written before keeping that in mind I have modified the code kindly review it

**Comment by mahek0620** [comment]
@aswinikalyan30 can you please review it

**Comment by aswinikalyan30** [comment]
Sure.

**Review by aswinikalyan30 — APPROVED** [review]
Great work!

**Pull request merged**